aikau/src/main/resources/alfresco/pickers/DocumentListPicker.js

```diff
@@ -133,8 +133,11 @@ define(["dojo/_base/declare",
        * @instance
        */
       postCreate: function alfresco_pickers_DocumentListPicker__postCreate() {
-         this.processObject(["processInstanceTokens"], this.widgets);
-         this.processWidgets(this.widgets, this.itemsNode);
+         // NOTE: Need to set viewModifiers for all subsequent calls to renderView...
+         this.viewModifiers =  ["processInstanceTokens"];
```

Comment:
Yes, DocumentList picker is a very specific implementation - without this modifier, it's not going to work.